tokio-process/Cargo.toml

```diff
@@ -17,19 +17,16 @@ An implementation of an asynchronous process management backed futures.
 categories = ["asynchronous"]
 
 [dependencies]
-futures = "0.1.11"
-tokio-io = "0.1"
-tokio-reactor = "0.1"
+futures-util-preview = { version = "0.3.0-alpha.17" }
+futures-preview = { version = "0.3.0-alpha.17" }
+tokio = { version = "0.2.0", path = "../tokio" }
```

Comment:
I think this should be a dev-dependency since we only need all of tokio for testing

Reply:
It's being used for `read_to_end` right now. Is there some replacement for `read_to_end` elsewhere.

Reply:
never mind that implementation comes from `futures_util::io` now.